File: wrappers/mvc/demos/Kendo.Mvc.Examples/Controllers/DataViz/Radar_Charts/Local_DataController.cs
using System;
using System.Linq;
using System.Web.Mvc;
using Kendo.Mvc.Examples.Models;

namespace Kendo.Mvc.Examples.Controllers
{
    public partial class Radar_ChartsController : Controller
    {
        public ActionResult Local_Data()
        {
            return View(ChartDataRepository.ProteinQualityData());
        }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/controllers/radarcharts/LocalDataController.java
package com.kendoui.spring.controllers.radarcharts;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

import com.kendoui.spring.models.ChartDataRepository;

@Controller("dataviz-radar_chart-local_data-controller")
@RequestMapping(value="/dataviz/radar-charts/")
public class LocalDataController {
    @RequestMapping(value = "/local-data", method = RequestMethod.GET)
    public String index(Model model) {
        model.addAttribute("proteinQuality", ChartDataRepository.ProteinQualityData());
     
        return "/dataviz/radar-charts/local-data";
    }
}
